cli/command/node/health.go

```diff
@@ -79,31 +75,16 @@ func runHealth(storageosCli *command.StorageOSCli, opt *healthOptions) error {
 
 // UpdateNodeHealth updates the health status of a given node by querying the
 // node endpoints.
-func UpdateNodeHealth(node *cliTypes.Node, address string, timeout int) error {
-	healthEndpointFormat := "http://%s:%s/v1/" + api.HealthAPIPrefix
-
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*time.Duration(timeout))
-	defer cancel()
-
-	client := &http.Client{}
-
-	var healthStatus types.HealthStatus
-	cpURL := fmt.Sprintf(healthEndpointFormat, address, api.DefaultPort)
-	cpReq, err := http.NewRequest("GET", cpURL, nil)
+func UpdateNodeHealth(storageosCli *command.StorageOSCli, node *cliTypes.Node) error {
+	cpHealth, err := storageosCli.Client().CPHealth(context.Background(), node.AdvertiseAddress)
```

Comment:
This function signature has removed the timeout which is now ignored when passed by the user. We should set a deadline on the context:

```suggestion
	ctx, cancel := context.WithTimeout(context.Background(), timeout)
	defer cancel()
	
	cpHealth, err := storageosCli.Client().CPHealth(context.Background(), node.AdvertiseAddress)
```